packages/vue/src/components/BaseRadioGroup/BaseRadioGroup.vue

```diff
@@ -43,7 +43,7 @@
           >
             <span
               v-if="option.title"
-              class="font-semibold"
+              :class="option.text ? 'font-semibold' : ''"
```

Comment:
I added this change directly since I couldn't comment on a file you hadn't modified. LMK if you have any concerns with it.

Reply:
good change, thank you!